//
// Created by viewsharp on 27.04.18.
//

#ifndef SERVER_CLIENT_H
#define SERVER_CLIENT_H

#include <service.pb.h>

class Client {
public:
    Client() {

    }

    void Queue(QueueReply queueReply) {

    }

    void Start(StartReply startReply) {

    }

    void Iteration(IterationReply iterationReply) {

    }

    void End(EndReply endReply) {

    }
};

#endif //SERVER_CLIENT_H
